code/__DEFINES/typeids.dm

```diff
@@ -3,6 +3,6 @@
 #define TYPEID_NORMAL_LIST "f"
 //helper macros
 #define GET_TYPEID(ref) ( ( (length(ref) <= 10) ? "TYPEID_NULL" : copytext(ref, 4, -7) ) )
-#define IS_NORMAL_LIST(L) (GET_TYPEID("\ref[L]") == TYPEID_NORMAL_LIST)
+#define IS_NORMAL_LIST(L) (GET_TYPEID(text_ref(L)) == TYPEID_NORMAL_LIST)
```

Comment:
given this should only be used on lists (its used to figure out if something is a normal list or a byond special list for vv) you can move this to use islist first since it should return false on non-lists.

it will be slightly faster but this is also vv code so it doesn't really matter much

Reply:
I'll just keep it as is pre change. this is actually a bit hot for reftracking, so I'd like to keep it cheap if possible